Bio/PDB/parse_pdb_header.py

```diff
@@ -219,6 +236,7 @@ def _parse_remark_465(line):
 
 def _parse_pdb_header_list(header):
     # database fields
+    # added ssbond 11/20/22 Eric G. Suchanek
```

Comment:
I'd remove these comments. Git will track these changes and keep your authorship intact if anyone wishes to know who wrote this code.